src/main/java/net/dv8tion/jda/api/requests/restaction/pagination/EntitlementPaginationAction.java

```diff
@@ -0,0 +1,89 @@
+package net.dv8tion.jda.api.requests.restaction.pagination;
+
+import net.dv8tion.jda.api.entities.Entitlement;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.entities.UserSnowflake;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * {@link PaginationAction PaginationAction} that paginates the application entitlements endpoint.
```

Comment:
Add a description note here that this also includes ended entitlements by default and how to disable that